src/nootils.js

```diff
@@ -41,10 +41,11 @@ module.exports = function(settings) {
       return report && report.reported_date;
     },
 
-    getMostRecentReport: function(reports, form, fields) {
+    getMostRecentReport: function(reports, forms, fields) {
+      if (!Array.isArray(forms)) return lib.getMostRecentReport(reports, [forms], fields);
```

Comment:
A cleaner way to do this, that doesn't require recursion, and is safer from this point of view, is to do: 
```suggestion
      if (!Array.isArray(forms)) {
        forms = [forms];
      }
```